README.md

```diff
@@ -1,4 +1,958 @@
+
+    > **Caution:** While the default values in the downloaded environment files enable the ELEVATE-Project Application to operate, certain features may not function correctly or could be impaired unless the adopter-specific environment variables are properly configured.
+
+    > **Important:** As mentioned in the above linked document, the **User SignUp** functionality may be compromised if key environment variables are not set correctly during deployment. If you opt to skip this setup, consider using the sample user account generator detailed in the `Sample User Accounts Generation` section of this document.
+
+5. **Create Databases**
+
+    - **Ubuntu/Linux**
+        1. Download `create-databases.sh` Script File:
+            ```
+            curl -OJL https://github.com/ELEVATE-Project/mentoring/raw/master/documentation/2.6.1/native/scripts/linux/create-databases.sh
+            ```
+        2. Make the executable by running the following command:
+            ```
+            chmod +x create-databases.sh
+            ```
+        3. Run the script file:
+            ```
+            ./create-databases.sh
+            ```
+
+6. **Run Migrations To Create Tables**
+
+    - **Ubuntu/Linux/MacOS**
+
+        1. Install Sequelize-cli globally:
+            ```
+            sudo npm i sequelize-cli -g
+            ```
+        2. Run Migrations:
+            ```
+            cd user/src && npx sequelize-cli db:migrate && cd ../.. && \
+            cd notification/src && npx sequelize-cli db:migrate && cd ../..
+            ```
+
+7. **Insert Initial Data**
+   Use ELEVATE-Project scripts to insert the initial data.
+
+    - **Ubuntu/Linux/MacOS**
+
+        ```
+        cd ELEVATE-Project/project-service/documentation/1.0.0/native/scripts/linux && \
+        sh entity-project-sample-data.sh && \
+        cd ~/ELEVATE-Project/ && \
+        cd user/src && npm run db:seed:all && cd ../..
+        ```
+
+8. **Start The Services**
+
+    Following the steps given below, 2 instances of each ELEVATE-Project backend service will be deployed and be managed by PM2 process manager.
+
+    - **Ubuntu/Linux**
+
+        ```
+        cd project-service && pm2 start app.js -i 2 --name project-service && cd .. && \
+        cd entity-management/src && pm2 start app.js -i 2 --name entity-management && cd ../.. && \
+        cd user/src && pm2 start app.js -i 2 --name user && cd ../.. && \
+        cd notification/src && pm2 start app.js -i 2 --name notification && cd ../.. && \
+        cd interface-service/src && pm2 start app.js -i 2 --name interface && cd ../.. && \
+        cd scheduler/src && pm2 start app.js -i 2 --name scheduler && cd ../..
+        ```
+
+9. **Run Service Scripts**
+
+    - **Ubuntu/Linux/MacOS**
+
+        ```
+        cd user/src/scripts && node insertDefaultOrg.js && node viewsScript.js && \
+        node -r module-alias/register uploadSampleCSV.js && cd ../../..
+        ```
+
+10. **Start The Portal**
```

Comment:
@Prajwal17Tunerlabs native setup they are running portal using the repo. just like how we clone our services